Owlcat/simms/simms.py

```diff
@@ -83,6 +83,9 @@ def toList(string,delimiter=',',f0=False):
     if date is None:
         date = '%d/%d/%d'%(time.localtime()[:3])
 
+    if msname is None:
```

Comment:
Yeah, that is just extra logging information.

On Tue, Feb 17, 2015 at 10:00 AM, Gijs Molenaar <notifications@github.com>
wrote:

> In Owlcat/simms/simms.py
> <https://github.com/ska-sa/owlcat/pull/16#discussion_r24798457>:
>
> > @@ -83,6 +83,9 @@ def toList(string,delimiter=',',f0=False):
> >      if date is None:
> >          date = '%d/%d/%d'%(time.localtime()[:3])
> >
> > +    if msname is None:
>
> it also contains these additions, is that intentional?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ska-sa/owlcat/pull/16/files#r24798457>.
>